susemanager-utils/susemanager-sls/salt/channels/debiankeyring.sls

```diff
@@ -0,0 +1,7 @@
+{%- if pillar.get('mgr_metadata_signing_enabled', false) %}
+/usr/share/keyrings/mgr-archive-keyring.gpg:
+  file.managed:
+    - source:
+      - salt://gpg/mgr-keyring.gpg
```

Comment:
I think a list is not needed when you have only one file. I am even unsure if this is supported.

Reply:
Right, changed to use single value.